css/css-basics_social-card/css/styles.css

```diff
@@ -1,3 +1,22 @@
 * {
   box-sizing: border-box;
```

Comment:
If you add the properties margin and padding and assign them a value of 0 you can define all the further margins and paddings yourself:
margin: 0;
padding: 0;

Reply:
copy thanks!